ui/src/app/components/ct-available-banner-maybe.tsx

```diff
@@ -23,7 +27,8 @@ const CT_COOKIE_KEY = 'controlled-tier-available';
 
 const shouldShowBanner = (
   profile: Profile,
-  cdrVersionTiers: CdrVersionTier[]
+  cdrVersionTiers: CdrVersionTier[],
+  accessTiersVisibleToUsers: string[]
 ) => {
   const ct = cdrVersionTiers?.find(
```

Comment:
I know this isnt part of this PR, but i had to re-read this a lot of time to understand what ct is. So ct is true if the environment has a CT cdr else false ? If yes is it possible to rename it accordingly

Reply:
That's correct, and yes this is a bad name for it.  It's now `ctCdrVersions`.  Thanks!